### DISCUSSION TITLE
Endpoints of the images

### DISCUSSION THREAD

**Discussion by Ivann-2612** [directus/directus#10553] [Archive]
One question,when I upload an image in some content in the Directus ,how can I get the API endpoint to show in the Frontend cards or something else?

This is an endpoint in json file  :

```
"data": [
    {
      "id": 1,
      "status": "published",
      "sort": null,
      "user_created": "79dc08e2-1b16-486f-9d48-5008dd099f72",
      "date_created": "2021-12-16T11:51:38",
      "user_updated": "79dc08e2-1b16-486f-9d48-5008dd099f72",
      "date_updated": "2021-12-16T11:52:47",
      "image": "edc5117c-8466-4366-bacc-8ebe14f1d95d",    **###  ------------- I need this to be an images file with extension jpg,png....**
      "description": "\"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod 
      "title": "Post one"
    }
```

.This is probably the ID of that image what I upload in the Diectus

**Comment by rijkvanzanten** [comment]
Heya, the endpoint to download the file (or get a thumbnail) is `/assets/:id` (see https://docs.directus.io/reference/files/). That ID is the ID of the file (which is also what you're seeing in that relational `image` field you've shown. You can nest the files metadata in this response through [the `fields` parameter](https://docs.directus.io/reference/query/#fields).

**Reply by Ivann-2612** [reply]
As you see,this is my example of work
![Screenshot (231)](https://user-images.githubusercontent.com/60244706/146399958-d5ebba7e-c5fd-497a-bced-4f71be429c79.png)

**Reply by Ivann-2612** [reply]
With this endpoint http://localhost:8055/items/Posts I am getting context,with axios

**Reply by Ivann-2612** [reply]
And from this endpoints I am using data for my Frontend UI

**Reply by Ivann-2612** [reply]
This is ID an image endpoints,I need image with extensions,jpg,png...

**Comment by Ivann-2612** [comment]
I found the answer, thank you my friend

**Reply by marcelowa** [reply]
what is the answer ?

**Reply by yashsuhagiya** [reply]
I am also looking for an answer for the same question. If you can share your finding!
